base/constants.go

```diff
@@ -90,6 +90,30 @@ const (
 
 	// The limit in Couchbase Server for total system xattr size
 	couchbaseMaxSystemXattrSize = 1 * 1024 * 1024 // 1MB
+
+	//==== Sync Prefix Documents & Xattr====
+	SyncPrefix    = "_sync:"
+	SyncXattrName = "_sync"
+
+	AttPrefix              = SyncPrefix + "att:"
+	BackfillCompletePrefix = SyncPrefix + "backfill:complete:"
+	BackfillPendingPrefix  = SyncPrefix + "backfill:pending:"
+	DCPBackfillSeqs        = SyncPrefix + "dcp_backfill"
+	DCPCheckpointPrefix    = SyncPrefix + "dcp_ck:"
+	DCPCkPrefix            = SyncPrefix + "dcp_ck:"
+	KSyncDataKey           = SyncPrefix + "syncdata"
```

Comment:
Would be good to group keys and prefixes separately so they're not hidden in this list.